controller/OWNERS

```diff
@@ -1,7 +1,7 @@
 # The OWNERS file is used by prow to automatically merge approved PRs.
 
 approvers:
-- pkg-controller-writers
+- serving-writers
 
 reviewers:
-- pkg-controller-reviewers
+- serving-reviewers
```

Comment:
I'm hoping prow is smart enough to rotate through this list of folks for reviews and approvals vs. the larger pool at the top OWNERS files